ramenpos/RamenPos/View Controllers/Main/MainViewController+PrintStatusActions.swift

```diff
@@ -175,6 +175,7 @@ extension MainViewController {
             logMessage(String(format: "# CASHOUT: %.2f", Float(cashoutResponse.getCashoutAmount()) / 100.0))
             logMessage(String(format: "# BANKED NON-CASH AMOUNT: %.2f", Float(cashoutResponse.getBankNonCashAmount()) / 100.0))
             logMessage(String(format: "# BANKED CASH AMOUNT: %.2f", Float(cashoutResponse.getBankCashAmount()) / 100.0))
+            logMessage(String(format: "# SURCHARGE: %.2f", Float(cashoutResponse.getSurchargeAmount()) / 100.0))
```

Comment:
Typically financial amounts should be in a Double for accuracy but no big deal here it's just a sample :)

Reply:
Ok i got it. Thanks